vividus-plugin-accessibility/src/main/java/org/vividus/bdd/converter/ExamplesTableToAccessibilityCheckOptionsConverter.java

```diff
@@ -0,0 +1,151 @@
+import org.vividus.ui.context.UiContext;
+import org.vividus.ui.web.action.ICssSelectorFactory;
+
+public class ExamplesTableToAccessibilityCheckOptionsConverter
+    extends AbstractChainableParameterConverter<ExamplesTable, List<AccessibilityCheckOptions>>
+{
+    private static final String LEVEL = "level";
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(
+            ExamplesTableToAccessibilityCheckOptionsConverter.class);
+
+    private static final String STANDARD = "standard";
+
+    private final UiContext uiContext;
+    private final ICssSelectorFactory cssSelectorFactory;
+    private final ISearchActions searchActions;
+
+    public ExamplesTableToAccessibilityCheckOptionsConverter(UiContext uiContext,
+            ICssSelectorFactory cssSelectorFactory, ISearchActions searchActions)
+    {
+        this.uiContext = uiContext;
+        this.cssSelectorFactory = cssSelectorFactory;
+        this.searchActions = searchActions;
+    }
+
+    @Override
+    public List<AccessibilityCheckOptions> convertValue(ExamplesTable toConvert, Type type)
+    {
+        return toConvert.getRowsAsParameters()
+                        .stream()
+                        .map(this::toOptions)
+                        .collect(Collectors.toList());
+    }
+
+    private AccessibilityCheckOptions toOptions(Parameters row)
+    {
+        AccessibilityStandard standardParameter = row.valueAs(STANDARD, AccessibilityStandard.class, null);
+        checkNotNull(standardParameter, STANDARD);
+        ViolationLevel level = row.valueAs(LEVEL, ViolationLevel.class, null);
+        checkNotNull(level, LEVEL);
+        Set<Locator> locators = row.valueAs("elementsToCheck", new TypeLiteral<Set<Locator>>() { }.getType(), Set.of());
+        Set<Locator> elementsToIgnore = row.valueAs("elementsToIgnore", new TypeLiteral<Set<Locator>>() { }.getType(),
+                Set.of());
+        List<String> violationsToIgnore = row.valueAs("violationsToIgnore",
+                new TypeLiteral<List<String>>() { }.getType(), List.of());
+        List<String> violationsToCheck = row.valueAs("violationsToCheck",
+                new TypeLiteral<List<String>>() { }.getType(), null);
+        return createOptions(standardParameter, violationsToIgnore, violationsToCheck, locators, elementsToIgnore,
+                level);
+    }
+
+    private AccessibilityCheckOptions createOptions(AccessibilityStandard standard, List<String> violationsToIgnore,
+            List<String> violationsToCheck, Set<Locator> elementsToCheck,
+            Set<Locator> elementsToIgnore, ViolationLevel level)
+    {
+        List<String> ignore = new ArrayList<>(violationsToIgnore);
+        Stream.of(ViolationLevel.values())
+              .filter(l -> l.getCode() > level.getCode())
+              .map(ViolationLevel::toString)
+              .map(String::toLowerCase)
+              .forEach(ignore::add);
+        AccessibilityCheckOptions options = createOptions(standard, ignore);
```

Comment:
this object configuration looks a bit weird because some part of that you moved to a separate method and the rest you left as it is, it's ok if we configure particular unit, but in this case 'createOptions' looks too generic